shared/validation/languages/common.js

```diff
@@ -0,0 +1,36 @@
+var unicode = require('unicode-properties');
+var decode = require('urldecode');
+var striptags = require('striptags');
+
+function _strip_string(sentence) {
+    //letters
+    var allowed_categories = ["LC", "Ll", "Lm", "Lo", "Lt", "Lu"];
```

Comment:
I don't have much experience with these categories, can you open a discussion on https://discourse.mozilla.org/c/voice so we can make sure this works for all languages?